docs/ENVIRONMENT.md

```diff
@@ -615,6 +615,11 @@ SYSTEMD_HOME_DEBUG_SUFFIX=foo \
   there already exists at least one regular user on the system. If set to "0"
   will make the tool skip any such query.
 
+* `$SYSTEMD_HOME_DRY_RUN` – if set to "1" will make `homectl create` and
```

Comment:
should this maybe be `SYSTEMD_DRY_RUN` like we have a single SYSTEMD_LOG_LEVEL? Might be useful in more tools in the future

Reply:
I'd really prefer to have this a bit "focussed", i.e. bypassing homed operations are one thing, bypassing others is another. i think there's value in controlling this independently.